ProcessMaker/Http/Controllers/Api/ProcessRequestController.php

```diff
@@ -176,27 +178,35 @@ public function index(Request $request, $getTotal = false, User $user = null)
         } catch (QueryException $e) {
             throw $e;
             $rawMessage = $e->getMessage();
-            if (preg_match("/Column not found: 1054 (.*) in 'where clause'/", $rawMessage, $matches)) {
-                $message = $matches[1];
-            } else {
-                $message = $rawMessage;
-            }
+            $message = $this->getMatches($rawMessage, $matches);
 
             return response(['message' => $message], 400);
         }
 
-        if (isset($response)) {
+        $response = isset($response)
             // Map each item through its resource
```

Comment:
Please, let's use the if structure. In the future, we may need to add other columns, so future changes will be clearer and easier to understand.